### ISSUE TITLE
make it work with exotic port numbers

### ISSUE THREAD

**Issue by Kranf99** [Kranf99/kibella#66]
Currently, Kibella works perfectly on the "standard" port numbers: i.e. port 80 for http and port 443 for https. ...But, if you are using any other port than the "standard" ones it fails.

More details: 
You can see here that the connection to "http://localhost/kibella/db/?_=1578..." is not working:
![2020-01-10_011045](https://user-images.githubusercontent.com/20713234/72115274-c4c20b80-3346-11ea-94a6-f67034a8b0ff.png)
...and this was to be expected because, in this example, kibella was installed on "http://localhost:4443". 

Thus, you can manually edit the url (to add the port number) and you can see that it's working (i.e. "status code" is OK):
![2020-01-10_011055](https://user-images.githubusercontent.com/20713234/72115372-179bc300-3347-11ea-9e92-cf732717c48e.png)

To summarize: Kibella "loses" the port-number for some requests to the back-end,
Would it be possible to fix that?

**Comment by Kranf99** [comment]
This is very closely related to issue #59.
When #59 is fixed, this one should also be fixed.

**Comment by Kranf99** [comment]
This appears to be fixed!
